.github/workflows/lint.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# From
+# https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/modules/lint_file/lint.yaml.tftpl
+
+name: 'lint'
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: 'lint'
+    runs-on: 'ubuntu-latest'
+    env:
+      EXCLUDE_LINT_DIRS: "\\./anthos-bm-openstack-terraform/resources"
```

Comment:
**Question:**
Is there an environment variable that lets us ignore certain rules?
I was wondering if we could ignore [SC2181](https://www.shellcheck.net/wiki/SC2181), which seems overkill for long commands such as [this long command](https://github.com/GoogleCloudPlatform/anthos-samples/blob/main/anthos-bm-openstack-terraform/resources/abm_setup_gcp.sh#L24) which the linter is complaining about. Related issue: https://github.com/koalaman/shellcheck/issues/2465.

Reply:
These can now be passed in: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/locals.tf